data/projects.yml

```diff
@@ -0,0 +1,16 @@
+title: Projects
```

Comment:
We could make this more explicit, also looks nicer on the website

```suggestion
title: OpenRail Projects
```

Reply:
Ok, then we should probably also change the title of the member section from "Membership" to "OpenRail Members".

Reply:
Good idea, will do it separately